authoraffsrv/views.py

```diff
@@ -138,7 +142,7 @@ def __export_to_excel(self):
         filename = self.TMP_EXCEL_FOLDER + self.EXPORT_FILENAME + str(uuid.uuid4())
         wbk.save(filename)
         # now read it
-        xls_file = open(filename)
+        xls_file = open(filename, encoding='ISO-8859-1')
```

Comment:
I'm not super familiar, but any reason not to use UTF-8? 
I think 8859 is pretty old if I'm not mistaken.

Reply:
as per your suggestion removed the encoding. apparently if you open it in byte mode you do not need the encoding. going to see if it works live. thank you for your suggestion.